logic.js

```diff
@@ -17,4 +19,20 @@ const generateNextWeek = day => {
   return result
 }
 
-module.exports = { daysUntilMonday, generateNextWeek };
+const formateDate = input => {
+  const regex = /^([0-9]+\.[0-9]+(\.)?)$/
+  if (!regex.test(input)) return DateTime.fromObject({ day: 0 })
+  const pieces = input.split(".")
+  let date = DateTime.fromObject({ month: pieces[1],  day: pieces[0] })
+  let now = DateTime.now()
+  if (date < now) date = date.plus({ years: 1 })
```

Comment:
kun date on tänään, tämä ei annakaan tämän päivän tietoja vaan hyppää ensi vuoteen. näyttäisi korjautuvan näin: `date < now.minus({ days: 1 })`. ehkä voisi samalla myös sallia tietojen saamisen vielä kuukauden tai 30 päivän takaa

Reply:
Hyvä kun huomasit tämän! :D Korjataan tämä, olisi pitänyt huomata kokeilla saman päivän tietojen saamista tosiaan :)